src/httpserver/http_utils.hpp

```diff
@@ -310,6 +320,11 @@ class arg_comparator {
      }
 };
 
+using value_map = std::map<std::string, std::string, http::header_comparator>;
+using arg_map = std::map<std::string, std::string, http::arg_comparator>;
+using value_map_view = std::map<std::string_view, std::string_view, http::header_comparator>;
+using arg_map_view = std::map<std::string_view, std::string_view, http::arg_comparator>;
```

Comment:
I am not sure I'd keep these - the maps aren't used enough around that it is worth it.

Reply:
There are ~52 instances of these types throughout the codebase, I typically use aliases for less. These kinds of template decls are super verbose as well :smile:

Reply:
That's a lot more than I thought. Fair to keep them - I'll open an issue for a general cleanup

Reply:
Just fix the names for clarity:

- value_map -> header_map
- value_map_view -> header_view_map
- arg_map is fine
- arg_map_view -> arg_view_map